benchmarks/Microsoft.AspNetCore.Routing.Performance/RouteParameterParserBenchmark.cs

```diff
@@ -0,0 +1,26 @@
+﻿using BenchmarkDotNet.Attributes;
+using Microsoft.AspNetCore.Routing.Patterns;
+
+namespace Microsoft.AspNetCore.Routing
+{
+    public class RouteParameterParserBenchmark
```

Comment:
Let me know if you want me to remove the benchmark.

Reply:
Yes remove them